src/utils/constants.js

```diff
@@ -49,7 +49,6 @@ export const FILE_STATUSES = {
   ERROR_DEFINITION: 'ERROR_DEFINITION',
   DELETING: 'DELETING',
   DISCARDED: 'DISCARDED',
-  CANCELLED: 'CANCELLED',
```

Comment:
Is there a particular reason why you are removing this line? Because these change was made recently.